draft-ietf-ohai-ohttp.md

```diff
@@ -683,10 +684,16 @@ indicating the content type, and the encapsulated response as the response
 content.  As with requests, additional fields MAY be used to convey information
 that does not reveal information about the encapsulated response.
 
+An Oblivious Gateway Resource that does not successfully receive a response can
```

Comment:
"successfully receives a response" doesn't reads a little odd to me (if a target response is failure, that's still success, it think...). What about changing L679 to say

> An Oblivious Gateway Resource that receives any response from the Target

And then make this line consistent and opposite with

> An Oblivious Gateway Resource that does not receive a response can itself